cscs-checks/system/openstack/s3_check.py

```diff
@@ -0,0 +1,132 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class S3apiCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        super().__init__()
+        endpoint = 'object.cscs.ch'
+        self.descr = 'S3API check for (%s)' % endpoint
+        self.tags = {'ops', endpoint}
+        self.valid_systems = ['dom:gpu', 'daint:gpu', 'daint:mc']
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        # self.sourcesdir = None
+        self.time_limit = (0, 5, 0)
+        self.maintainers = ['VH', 'GLR']
+        self.executable = 's3_test.sh'
+        if self.current_system.name in ['dom']:
+            self.tags = {'production'}
+
+@rfm.simple_test
+class S3apiCreateBucket(S3apiCheck):
+    def __init__(self):
+        super().__init__()
+        self.executable_opts = ['s3_create_bucket.py',
+                                self.current_system.name]
+
+        self.sanity_patterns = sn.assert_found(r'Average bucket creation time',
+                                               self.stdout)
+        self.perf_patterns = {
+            'bucket_ctime': sn.extractsingle(
+                r'Average bucket creation time \(s\):\s(?P<secs>\S+)',
+                self.stdout, 'secs', float)
+        }
+
+        self.reference = {
+            '*': {
+                # below 1.1 secs is ok
+                'bucket_ctime': (1, None, 0.01),
```

Comment:
Put units to all your reference numbers.

```suggestion
                'bucket_ctime': (1, None, 0.01, 's'),
```